src/app/icons/icons-sets/icons-sets.component.ts

```diff
@@ -182,12 +184,24 @@ export class IconsSetsComponent implements AfterViewInit, OnDestroy {
             })
         );
 
+        // send GA searched icons
+        // debounceTime is 1 second
+        this.subscriptions.push(this._iconsViewService.searchValue
+            .debounceTime(1000)
```

Comment:
@mathisscott We are pinging our GA here.

Reply:
I needed to ping our GA in its own independent subscription that has a `.debounceTime(1000)`. Otherwise, we might pollute the GA report with too many incomplete search terms.